spec/index.bs

```diff
@@ -5888,6 +5889,19 @@ dictionary GPUImageCopyTextureTagged : GPUImageCopyTexture {
         2d canvases are [[html#premultiplied-alpha-and-the-2d-rendering-context|always premultiplied]],
         while WebGL canvases can be controlled via <l spec=html>[=WebGLContextAttributes=]</l>.
         {{ImageBitmap}} premultiplication can be controlled via {{ImageBitmapOptions}}.
+
+    : <dfn>originBottomLeft</dfn>
+    ::
+        Describes whether the origin (0, 0) to copy from source image is bottom left corner, or not.
+       
+        If this option is set to `true` and the origin (0, 0) of {{GPUImageCopyExternalImage/source}} is top left corner,
+        the source values must be flipped along the vertical axis to change the origin (0, 0) to bottom left corner.
+
```

Comment:
I think we also need a clarification about whether this changes the `GPUImageCopyExternalTexture.origin` to be relative to the bottom-left corner (increasing upward) (`origin: [1, 0]` would cut off the bottom of the image), or if it is actually in the coordinate space of the source image (`origin: [1, 0]` would cut off the top of the image).

Naming-wise, I think:
`originBottomLeft` strongly implies that `origin` is relative to the bottom left (because `.originBottomLeft` seems to refer to `.origin`).
`source.flipY` would weakly imply that it's still relative to the top left.
`destination.flipY` would strongly imply that it's still relative to the top left.

TBH I have no idea which is better.

Reply:
Ref to [flipY](https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#dom-imageorientation-flipy) in ImageBItmap, it has a more simple defination. It says, "_output must be flipped vertically, disregarding any image orientation metadata of the source (such as EXIF metadata), if any. "

Do you think we could adopt this description so that we can ignore the origin corner issue. (BTW, I think WebGL has the same meaning for [UNPACK_FLIP_Y_WEBGL](https://www.khronos.org/registry/webgl/specs/latest/1.0/#6.10))

Reply:
AFAIK none of our source types can have orientation metadata, but it wouldn't hurt to be explicit about it anyway. (I am surprised that createImageBitmap ignores EXIF metadata though. I noticed that earlier too. Doesn't matter for us now since we don't have HTMLImageElement sources.)

I'd be fine with flipY, I don't have strong feelings, but I still slightly prefer originBottomLeft because I think it's less ambiguous. Either way will require a clarification about what `origin` means. And either way developers will figure it out without much trouble.

Reply:
Thanks! I'll adopt the `originBottomLeft`, move it to `GPUImageCopyExternalTexture` and update the words.